backend/src/main/java/com/crm/dto/response/AccountResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.crm.dto.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Builder
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class AccountResponse {
```

Comment:
Czy podczas tworzenia konta jest to nam niezbędne? Chyba wystarczy tylko kod 200.

Reply:
Właśnie zastanwiałem się co zwracać podczas stworzenia konta. Zrobię jak napisałeś

Reply:
done